### PR TITLE
Transition contributions and offers flow to on() and execute() APIs

### PR DESCRIPTION
**Pull request by sohanirao**

### CODE REVIEW

File: src/proto/api_messages_test.js [hunk 1]
```diff
@@ -56,6 +60,20 @@ function isEqual(thisArray, otherArray) {
 
 describe('api_messages', () => {
 
+  describe('test_AlreadySubscribedResponse', () => {
+    it('should deserialize correctly', () => {
+      const /** !AlreadySubscribedResponse  */ alreadysubscribedresponse = new AlreadySubscribedResponse();
```
Comment (ChrisAntaki) [inline src/proto/api_messages_test.js]: Would it be easy to make these local test variables camelCase?

File: src/runtime/contributions-flow-test.js [hunk 2]
```diff
@@ -29,7 +33,7 @@ describes.realWin('ContributionsFlow', {}, env => {
   let callbacksMock;
   let pageConfig;
   let port;
-  let messageCallback;
+  const messageMap = {};
```
Comment (ChrisAntaki) [inline src/runtime/contributions-flow-test.js]: This is a cool pattern, but I worry it leaks state across tests. Could the messageMap be declared with `let` instead, and then cleared `beforeEach` test?

File: src/proto/api_messages_test.js (hunk 1 above)
Comment (ChrisAntaki) [reply src/proto/api_messages_test.js]: This comment is more of a nit, for readability

Comment (sohanirao) [reply src/proto/api_messages_test.js]: Sure please feel free to bug on Protobuf Library generator code and assign to me. Would be happy to address it during fix it.

File: src/runtime/contributions-flow-test.js (hunk 2 above)
Comment (sohanirao) [reply src/runtime/contributions-flow-test.js]: good idea. done

File: src/proto/api_messages_test.js (hunk 1 above)
Comment (ChrisAntaki) [reply src/proto/api_messages_test.js]: Sounds good

**Comment by sohanirao** [comment]
@googlebot I fixed it.

**Comment by sohanirao** [comment]
This PR was actually submitted using P#646

**Pull request closed without merging**